src/NDS.cpp

```diff
@@ -1916,7 +1947,7 @@ u16 ARM9Read16(u32 addr)
 
     switch (addr & 0xFF000000)
     {
-    case 0x02000000:
+    case 0x02000000:        
```

Comment:
There seems to be extra whitespace at the end of this line.

Reply:
Thanks for reminding me, WaluigiWare64! I'll fix it as soon as I can!